module/bdev/lvol/vbdev_lvol.c

```diff
@@ -934,43 +1050,146 @@ lvol_write(struct spdk_lvol *lvol, struct spdk_io_channel *ch, struct spdk_bdev_
 static int
 lvol_reset(struct spdk_bdev_io *bdev_io)
 {
-	spdk_bdev_io_complete(bdev_io, SPDK_BDEV_IO_STATUS_FAILED);
+	lvol_op_comp(bdev_io, -EPERM);
```

Comment:
Why change it to EPERM?

Reply:
If you pass an ERRNO different from `ENOMEM` to `lvol_op_comp`, it will call `spdk_bdev_io_complete` with `SPDK_BDEV_IO_STATUS_FAILED`. So nothing changes.